tests/components/mqtt/common.py

```diff
@@ -66,6 +66,254 @@
     "configuration_url": "http://example.com",
 }
 
+MOCK_SUBENTRY_NOTIFY_COMPONENT1 = {
+    "notify_milkman_alert": {
+        "platform": "notify",
+        "name": "Milkman alert",
+        "qos": 0,
+        "command_topic": "test-topic",
+        "command_template": "{{ value_json.value }}",
+        "entity_picture": "https://example.com/notify_milkman_alert",
+        "retain": False,
+    },
+}
+MOCK_SUBENTRY_NOTIFY_COMPONENT2 = {
+    "notify_the_second_notifier": {
+        "platform": "notify",
+        "name": "The second notifier",
+        "qos": 0,
+        "command_topic": "test-topic2",
+        "entity_picture": "https://example.com/notify_the_second_notifier",
+    },
+}
+MOCK_SUBENTRY_NOTIFY_COMPONENT_NO_NAME = {
+    "notify_none": {
+        "platform": "notify",
+        "qos": 0,
+        "command_topic": "test-topic",
+        "command_template": "{{ value_json.value }}",
+        "entity_picture": "https://example.com/notify_none",
+        "retain": False,
+    },
+}
+
+MOCK_SUBENTRY_SENSOR_COMPONENT = {
+    "sensor_bla789": {
+        "platform": "sensor",
+        "name": "Test sensor",
+        "qos": 1,
+        "state_topic": "test-topic3",
+        "entity_picture": "https://example.com/sensor_bla789",
+    },
+}
+
+MOCK_SUBENTRY_LIGHT_COMPONENT = {
+    "light_bla0ab": {
+        "platform": "light",
+        "name": "Test light",
+        "qos": 1,
+        "command_topic": "test-topic4",
+        "schema": "basic",
+        "entity_picture": "https://example.com/light_bla0ab",
+    },
+}
```

Comment:
Isn't it a little bit weird to introduce things from the future like this? 🤔
Maybe in this PR do two notify entities, and then replace with or add other entities when they are supported?